modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replication/request/ReplicaRequestParameters.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.table.distributed.replication.request;
+
+import java.util.Collection;
+import org.apache.ignite.internal.schema.BinaryRow;
+import org.apache.ignite.internal.tx.InternalTransaction;
+import org.apache.ignite.network.ClusterNode;
+
+/**
+ * Parameters for replica request.
+ */
+public class ReplicaRequestParameters {
```

Comment:
Where is the class use?

Reply:
ReplicaRequestParameters is used for creating replica request in InternalTableImpl#enlistInTx. May be it is not the best solution. I think it better than creating FiveFunction (previously there was a TriFunction).